src/include/OSL/llvm_util.h

```diff
@@ -49,7 +50,7 @@ namespace llvm {
 OSL_NAMESPACE_ENTER
 
 namespace pvt {   // OSL::pvt
-
+class ShaderInstance;
```

Comment:
Is this forward declaration used? I thought `llvm_util.h` was trying to stay mostly agnostic about OSL specifics.

Reply:
Doesn't appear its used.